build/bin/entrypoint

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+# This is documented here:
+# https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines
+
+if ! whoami &>/dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "${USER_NAME:-mattermost-cloud}:x:$(id -u):$(id -g):${USER_NAME:-mattermost-cloud} user:${HOME}:/sbin/nologin" >> /etc/passwd
+  fi
+fi
+
+exec ${CLOUD} server $@
```

Comment:
We will need to change the provisioning server to accept env vars for server config parameters. Right now, there is a required flag that needs to be set here: `--route53-id`

Reply:
we can set that in the args
but will be better and easy if accept env vars